<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="3118" started-at="2014-04-18T15:20:07Z" finished-at="2014-04-18T15:20:10Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3118" started-at="2014-04-18T15:20:07Z" finished-at="2014-04-18T15:20:10Z">
      <class name="com.example.tests.TestFolderCreation">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.TestFolderCreation@ba41381]" name="setUp" is-config="true" duration-ms="9" started-at="2014-04-18T18:20:07Z" finished-at="2014-04-18T18:20:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testFoldersWithSameNameCreation()[pri:0, instance:com.example.tests.TestFolderCreation@ba41381]" name="testFoldersWithSameNameCreation" duration-ms="3090" started-at="2014-04-18T18:20:07Z" finished-at="2014-04-18T18:20:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "Duplicated folder name"
     but: was null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "Duplicated folder name"
     but: was null
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at com.example.tests.TestFolderCreation.testFoldersWithSameNameCreation(TestFolderCreation.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFoldersWithSameNameCreation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.TestFolderCreation@ba41381]" name="tearDown" is-config="true" duration-ms="1" started-at="2014-04-18T18:20:10Z" finished-at="2014-04-18T18:20:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.TestFolderCreation -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
